cryptoservice/import_export.go

```diff
@@ -99,18 +104,45 @@ func (cs *CryptoService) ExportRootKeyReencrypt(dest io.Writer, keyID string, ne
 // It prompts for the key's passphrase to verify the data and to determine
 // the key ID.
 func (cs *CryptoService) ImportRootKey(source io.Reader) error {
+	return cs.ImportRoleKey(source, data.CanonicalRootRole, nil)
+}
+
+// ImportRoleKey imports a private key in PEM format key from an io.Reader
+// It prompts for the key's passphrase to verify the data and to determine
+// the key ID.
+func (cs *CryptoService) ImportRoleKey(source io.Reader, role string, newPassphraseRetriever passphrase.Retriever) error {
```

Comment:
I'm fine with either changing to `ImportRoleKey` or just not even calling `ImportRootKey` from the `key import` command.  It might be cleaner just to change the interface, though it'll make this PR a little bigger